lib/collection/property-list.js

```diff
@@ -229,25 +235,29 @@ _.extend(PropertyList.prototype, /** @lends PropertyList.prototype */ {
      * Iterate on each item of this list
      */
     each: function (iterator, context) {
-        _.each(this.members, iterator, context || this.__parent);
+        _.forEach(this.members, iterator.bind(context || this));
```

Comment:
This is okay. I am not sure the old code was correct. We should keep the old code and add add a todo saying to check whether it's a bug in old code.